### PR TITLE
feat(configuration): [saleor] add support to save two different configs

### PR DESCRIPTION
**Pull request by AkshayaFoiger**
Extending the plugin to support both Hyperswitch and Juspay configuration
Screenshot 2024-08-01 at 2 59 00 PM

### CODE REVIEW

File: src/modules/ui/organisms/AddHyperswitchConfigurationForm/AddJuspayCredentialsForm.tsx
Comment (mrudulvajpayee4935) [inline src/modules/ui/organisms/AddHyperswitchConfigurationForm/AddJuspayCredentialsForm.tsx:1]: add field for merchantId too. We will need this

File: src/modules/hyperswitch/hyperswitch-api.ts [hunk 1]
```diff
@@ -10,19 +15,20 @@ import {
...
   PaymentAppConfigEntryFullyConfigured,
   paymentAppFullyConfiguredEntrySchema,
-} from "../payment-app-configuration/config-entry";
+} from "../payment-app-configuration/common-app-configuration/config-entry";
+import { config } from "../../pages/api/webhooks/saleor/transaction-initialize-session";
+import { HyperswitchFullyConfiguredEntry } from "../payment-app-configuration/hyperswitch-app-configuration/config-entry";
+import { invariant } from "@/lib/invariant";
 
 const SANDBOX_BASE_URL: string = "https://sandbox.hyperswitch.io";
 const PROD_BASE_URL: string = "https://api.hyperswitch.io";
```
Comment (mrudulvajpayee4935) [inline src/modules/hyperswitch/hyperswitch-api.ts:24]: should we store this as env variables?

File: src/modules/payment-app-configuration/juspay-app-configuration/config-entry.ts
Comment (mrudulvajpayee4935) [inline src/modules/payment-app-configuration/juspay-app-configuration/config-entry.ts:1]: need to store juspay merchantId in config

File: src/modules/hyperswitch/hyperswitch-api.ts (hunk 1 above)
Comment (AkshayaFoiger) [reply src/modules/hyperswitch/hyperswitch-api.ts:24]: yes, updated the env variables

**Pull request merged**